Add slot for toolbar (#2857)
ni/nimble: packages/spright-components/src/chat/specs/README.md

1. Lays out messages vertically based on their order.
1. Displays a vertical scrollbar if there are more messages than fit in the height allocated to the conversation.

> 1. Includes a slot to place toolbar content (such as buttons or menu buttons) on top of the conversation.

Reviewer: Similar feedback as I left on the story: we should make it clear that the client provides the toolbar and the content, not just the content.
```suggestion
1. Includes a slot to place a toolbar (and its content such as buttons or menu buttons) on top of the conversation.
```

    - The conversation will have a minimum width that clients are discouraged from overriding.
- _Slots_
    - chat messages are added to the default slot. The DOM order of the messages controls their screen order within the conversation (earlier DOM order => earlier message => top of the conversation)
    - a single chat input can optionally be added to the `input` slot. It will be placed below the messages

>     - optional slot for toolbar content (such as buttons or menu buttons) that will be displayed above the messages. This can be used for actions like starting a new conversation, copying all messages, or other conversation-level operations.

Reviewer: ```suggestion
    - a toolbar can optionally be added to the `toolbar` slot. The toolbar will be displayed above the messages. The toolbar content can be buttons or menus used for actions like starting a new conversation, copying all messages, or other conversation-level operations.
```
